build-system/tasks/report-test-status.js

```diff
@@ -25,13 +25,13 @@ const {isTravisPullRequestBuild} = require('../travis');
 const reportBaseUrl = 'https://amp-test-status-bot.appspot.com/v0/tests';
 
 const IS_GULP_E2E = argv._[0] === 'e2e';
-const IS_GULP_TEST = argv._[0] === 'test';
+const IS_GULP_TEST = argv._[0] === 'test' || argv._[0] === 'unit';
```

Comment:
Since the task name is being changed to `gulp unit`, maybe add `IS_GULP_UNIT` and simplify `inferTestType()`?

Reply:
I can clean this up when splitting out the integration test

Reply:
Actually nevermind, done